src/main/java/racingcar/controller/RaceController.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.controller;
+
+import java.util.List;
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+import racingcar.domain.RacingGame;
+import racingcar.domain.RacingGameConfig;
+import racingcar.view.InputView;
+import racingcar.view.ResultView;
+
+public class RaceController {
+    private final InputView inputView;
+    private final ResultView resultView;
+
+    public RaceController(final InputView inputView, final ResultView resultView) {
+        this.inputView = inputView;
+        this.resultView = resultView;
+    }
+
+    public void startRace() {
+        List<String> carNames = getCarNames();
+        int totalRounds = getTotalRounds();
+
+        RacingGameConfig gameConfig = new RacingGameConfig(carNames, totalRounds, resultView);
+        RacingGame racingGame = RacingGame.createWithRandomMove(gameConfig);
+        racingGame.play();
```

Comment:
이 부분은 함수화해서 호출될 수 있는 부분인 것 같습니다. 
아래에서 startRace()에 대한 개선 방법 언급이 나와서 말씀드려요 !

Reply:
```suggestion
        // 변수명 수정: totalRounds -> totalRound
        RacingGame racingGame = performRace(carNames, totalRound)
```
이렇게 할 수 있겠네요. 고민했던 부분인데 감사합니다!

Reply:
중간결과만 묶으면 최종결과를 위해 RacingGame을 반환해야 하니
최종결과까지 아래 코드 한 줄로 묶는 형태도 괜찮을 것 같습니다.
`performRace(carNames, totalRound);`

Reply:
개선한 최종본은 새 코멘트로 적었습니다.